src/PhpImap/Mailbox.php

```diff
@@ -326,7 +327,20 @@ public function setAttachmentsIgnore(bool $attachmentsIgnore): void
     {
         $this->attachmentsIgnore = $attachmentsIgnore;
     }
-
+	
+    /**
+     * Set $this->setAttachmentsRandomFilename param. 
+     *
+     * @param int $random ATTACH_FILE_NAMERANDOM, ATTACH_FILE_NAMEORIGINAL
```

Comment:
The variable in the comment should match the actual parameter variable.
```suggestion
     * @param int $mode ATTACH_FILE_NAMERANDOM, ATTACH_FILE_NAMEORIGINAL
```